.github/workflows/element-build-and-test.yaml

```diff
@@ -88,6 +88,19 @@ jobs:
             cypress/videos
             cypress/synapselogs
 
+      - name: Store benchmark result
+        uses: matrix-org/github-action-benchmark@jsperfentry-1
+        with:
+          name: Cypress measurements
+          tool: 'jsperformanceentry'
+          output-file-path: cypress/performance/measurements.json
+          # The dashboard is available at https://matrix-org.github.io/matrix-react-sdk/cypress/bench/
+          benchmark-data-dir-path: cypress/bench
+          fail-on-alert: false
+          comment-on-alert: false
```

Comment:
why not enable either of these?

Reply:
Could do, up to the team really. The current configuration is just copied from a previous usage of these benchmarking action. Perhaps it would be good to get a few data points in the dashboard and then define an appropriate threshold for alerting?